docs/main/eventing-communication/README.md

```diff
@@ -1,21 +1,48 @@
-# Using Knative Kafka broker
-If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+# Using Knative broker for eventing communication
 
-Kafka perfectly fullfills this reliability need.
+## Automated installation steps
+Usage:
+```
+Usage: ORCHESTRATOR_NAME=ORCHESTRATOR_NAME BROKER_NAME=BROKER_NAME BROKER_NAMESPACE=BROKER_NAMESPACE [KAFKA_REPLICATION_FACTOR=KAFKA_REPLICATION_FACTOR] [ORCHESTRATOR_NAMESPACE=openshift-operators] [BROKER_TYPE=kafka] [INSTALL_KAFKA_CLUSTER=true] ./eventing-automate-install.sh
```

Comment:
can you breakdown this line into multiple lines? it is rendered as a very long line.
In addition, the copy button will copy everything.